content/en/docs/concepts/containers/images.md

```diff
@@ -44,6 +44,26 @@ There are additional rules about where you can place the separator
 characters (`_`, `-`, and `.`) inside an image tag.  
 If you don't specify a tag, Kubernetes assumes you mean the tag `latest`.
 
+## Image digest
+
+An image [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier) is a 
+unique identifier for a specific version of a container image. 
+It is calculated using a hash function that takes account of the contents of all the layers in the image.
+
+The image digest is a fixed identifier that can't be changed, even if the image is updated. 
+
+An image digest is a long string of characters, typically in 
+the form of a [SHA-256](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#sha-256) 
+hash, this uniquely identifies a specific version of a container image.
+
+This is more secure and reliable way of referencing 
+container images because image digest is based on the contents of the image, 
+it provides a way to ensure that the image has not been tampered or modified with.
```

Comment:
```suggestion
it provides a way to ensure that the image has not been tampered with or modified.
```
'Modified with' doesn't sound right. what it is modified with? did you mean tampered with?